File: nvxapp.server.service/ClientServer-Service/Account/Models/CompanyListModel.cs
using nvxapp.server.data.Entities.Public;
using nvxapp.server.service.ClientServer_Service.ModelsBase;
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using static nvxapp.server.data.Entities.AspNetUsersDataUtil;

namespace nvxapp.server.service.ClientServer_Service.Account.Models
{
    public class CompanyListInModel
    {
    }

    public class CompanyListOutModel : ModelResult
    {
        public List<CompanyListModel> CompanyList { get; set; }

        public CompanyListOutModel()
        {
            CompanyList = new List<CompanyListModel>();
        }
    }

    public class CompanyListModel
    {
        public int IdCompany { get; set; }
        
        public required string IdAspNetUsers { get; set; }

        public string? Descrizione { get; set; }
        
    }

}

File: nvxapp.client/src/nvx/ClientServer-Service/Account/Models/company-list-model.ts
import { ModelResult } from "../../ModelsBase/model-result";
import { RolesModel } from "./user-roles-model";


export class CompanyListInModel {
  
}

export class CompanyListOutModel extends ModelResult {

  public companyList: CompanyListModel[] ;

}

export class CompanyListModel  {

  public idCompany: number;

  public descrizione: string | null;

  public idAspNetUsers: string ;

}

